### DISCUSSION TITLE
I want to be able to specify which dashboards are listed on the “Example” tab of the landing page

### DISCUSSION THREAD

**Discussion by EBoisseauSierra** [apache/superset#18544] [Ideas]
## Problem

I would like to curate my own Superset example dashboards (that would use “real” data and be user-specific), and that these dashboards appear in the “Example” tab in Superset landing page.

![Screenshot from 2022-01-10 12-50-39_shadow](https://user-images.githubusercontent.com/37387755/148768806-ff0858d8-e6e5-45a8-8ea7-cedd50b43384.png)

## Describe the solution you'd like

I would like to be able to define in a given dashboard settings, whether this dashboard is an Example dashboard or not — e.g. via a toggle switch.

One might want to protect this feature with a specific privilege (so that only a few people can define the Examples of all users of the Superset instance).

## Alternatives considered

As discussed [on Slack](https://apache-superset.slack.com/archives/C0170U650CQ/p1636630702324500), an “Example” dashboard is a dashboard which `created_by_fk` is `NULL`. My current workaround is to `UPDATE dashboards SET created_by_fk = NULL WHERE id = <my_dashboard_id>;` on the back-end database… but it's not exactly a good practice and I don't always have admin access to back-end dbs.

## Additional context

This question is related to (yet distinct from) #17408 : we're here talking about the end result, not the back-end implementation.

**Comment by geido** [comment]
I think the idea of expanding this concept and providing a way to categorize dashboards is the way to go. https://github.com/apache/superset/issues/17408#issuecomment-967008981

I can understand that being able to set a Dashboard as an example might be useful to some, but the level of abstraction that categorization would bring is a big big plus. The only downside is that it might come with some complexity and design considerations.

I believe the best approach to get attention and consensus is to write a SIP https://github.com/apache/superset/issues/5602. I can help with that @EBoisseauSierra

**Comment by geido** [comment]
For reference, I think categorization might help satisfying the requirement shared in this other issue https://github.com/apache/superset/issues/18005

**Comment by eschutho** [comment]
@betodealmeida implemented something with customizing examples that may help here.

**Comment by Bottlecapps** [comment]
I am not sure if this the right place to bring this up but a business user one of the biggest issue we have is the option to customize the landing page. Like 90% of the users may only use one dashboard so why not let them configure it to be their landing page?

**Comment by geido** [comment]
Hey @Bottlecapps why not creating a new issue with more information about the feature and how you envision that?
